### PR TITLE
add aiuto as code owner. direwct chagnes to distribution to them

### PR DESCRIPTION
**Pull request by aiuto**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1,2 @@
-* @c-parsons @laurentlb @jin
+* @c-parsons @laurentlb @jin @aiuto
+distribution @aiuto @fwe
```
Comment (c-parsons) [inline CODEOWNERS]: For my own information, what's the significance of "distribution changes"?

Comment (thomasvl) [reply CODEOWNERS]: https://help.github.com/en/articles/about-code-owners

I'm guessing you want `distribution/*` instead.

Comment (c-parsons) [reply CODEOWNERS]: Ah so this is a directory-to-be?

Comment (c-parsons) [reply CODEOWNERS]: Nvm. I see the other PR.

Comment (c-parsons) [reply CODEOWNERS]: Err, approved after this is changed to distribution/* :)

Comment (aiuto) [reply CODEOWNERS]: Oh. duh. Yes. The empty directory shows up in my fork, so I don't notice it not being real yet.

Comment (aiuto) [reply CODEOWNERS]: It doesn't need the trailing *, just the /

**Comment by aiuto** [comment]
fixed the path.

**Pull request merged**